runtime/doc/eval.txt

```diff
@@ -2041,6 +2041,7 @@ garbagecollect([{atexit}])	none	free memory, breaking cyclic references
 get({list}, {idx} [, {def}])	any	get item {idx} from {list} or {def}
 get({dict}, {key} [, {def}])	any	get item {key} from {dict} or {def}
 get({func}, {what})		any	get property of funcref/partial {func}
+get_xdg_home({string})          String  get the XDG Base directory for {string}
```

Comment:
I'm referring to the characters `xdg`.  On windows it's `%APPDATA%` or `%LOCALAPPDATA%`.  `XDG_DATA_HOME ` could've been `My Documents` on Windows.  The relationship between `XDG_*` and the Windows equivalents are purely a "best fit" decided by us.  The point is, `xdg` has no meaning on Windows.

> I don't like config_path() because there is a data and cache directory too.

That's why I said it was up for debate :-)

My view is that it should be something that can be found using intuition.  The keywords `xdg` and `home` don't say "the place where program configs and data is stored" to me.